CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+0.35.0 (2021-08-02)
```

Comment:
changelog should contain everything that has changed since last release.
Please check commit history and add to the changelog the missing changes